README.md

```diff
@@ -29,48 +30,78 @@ juju deploy postgresql --channel edge -n <number_of_units>
 
 To retrieve primary replica one can use the action `get-primary` on any of the units running PostgreSQL.
 ```shell
-juju run-action postgresql/<unit_number> get-primary --wait
+juju run-action postgresql/leader get-primary --wait
 ```
 
+Similarly, the primary replica is displayed as a status message in `juju status`, however one
+should note that this hook gets called on regular time intervals and the primary may be outdated if
+the status hook has not been called recently.
+
 ### Replication
 #### Adding Replicas
+
 To add more replicas one can use the `juju add-unit` functionality i.e.
 ```shell
 juju add-unit postgresql -n <number_of_units_to_add>
 ```
 The implementation of `add-unit` allows the operator to add more than one unit, but functions internally by adding one replica at a time, avoiding multiple replicas syncing from the primary at the same time.
 
 #### Removing Replicas
+
 Similarly to scale down the number of replicas the `juju remove-unit` functionality may be used i.e.
 ```shell
 juju remove-unit postgresql <name_of_unit1> <name_of_unit2>
 ```
 The implementation of `remove-unit` allows the operator to remove more than one unit. The functionality of `remove-unit` functions by removing one replica at a time to avoid downtime.
 
 ### Password rotation
+
 #### Charm users
+
 For users used internally by the Charmed PostgreSQL Operator an action can be used to rotate their passwords.
 ```shell
-juju run-action postgresql/0 set-password username=<username> password=<password> --wait
+juju run-action postgresql/leader set-password username=<username> password=<password> --wait
 ```
-Currently, the users used by the operator are `operator` and `replication`. Those users should not be used outside the operator.
+Note: currently, the users used by the operator are `operator` and `replication`. Those users should not be used outside the operator.
```

Comment:
We also have a `rewind` user
```suggestion
Note: currently, the users used by the operator are `operator`, `rewind` and `replication`. Those users should not be used outside the operator.
```

Reply:
Charm also has `backup` user. Should we expose the whole list?
It it actually spotted automatically:
```
juju run-action postgresql/leader get-password username=blablabla --wait
...
  message: 'The action can be run only for users used by the charm or Patroni: backup,
    replication, rewind, operator not blablabla'
```

Reply:
I think it's good to state that none of them should be used outside the charm. Other charms and applications should use only users created through relations. The `operator` user is a superuser and we should avoid using them as per https://patroni.readthedocs.io/en/latest/README.html#applications-should-not-use-superusers. (I know that we can bypass that with relation roles, but it's better to avoid using the charm users).

Reply:
Add all users.